josm-presets/de-signals-bostrab.xml

```diff
@@ -1586,6 +1586,110 @@ See https://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
 					de.display_values="Mastbauform,Zwergsignal"
 					/>
 			</item>
+			<item name="Combined Signal (H)" de.name="Mehrabschnittssignal (H)" icon="de/hha/h0-32.png" type="node">
+				<label text="Combined Signal (H)" de.text="Mehrabschnittssignal (H)" />
+				<label text="Only on VBK infrastructure (in Stadtbahntunnel Karlsruhe)" de.text="Nur auf VBK-Infrastruktur (im Stadtbahntunnel Karlsruhe)" />
+				<label text="Is being mapped as a node on the track." de.text="Wird als Punkt auf dem Gleis erfasst." />
+				<space />
+				<reference ref="sig_ref" />
+				<check key="railway:signal:combined:deactivated"
+					text="Out of order"
+					de.text="Ungültigkeitskreuz"
+					/>
+				<combo key="railway:signal:position"
+					text="Signal position"
+					de.text="Signalstandort"
+					values="left,right,bridge,overhead"
+					de.display_values="Links von Wegrichtung,Rechts von Wegrichtung,Signalbrücke,Oberleitung"
+					display_values="left in direction of OSM way,right in direction of OSM way,bridge,catenary"
+					/>
+				<reference ref="direction_fw_bw" />
+				<space />
+				<key key="railway:signal:combined"
+					value="DE-BOStrab:h" />
+				<multiselect key="railway:signal:combined:states"
+					text="Possible Signals"
+					de.text="Anzeigbare Signale"
+					delimiter=";"
+					values="DE-VBK:h0;DE-VBK:h1;DE-VBK:h1b;DE-VBK:h2;DE-VBK:h3"
+					display_values="H 0 (Stop, red);H 1 (Proceed, green);H 1 b (Proceed and expect speed limit, blinking green);H 2 (Proceed, expect stop on next signal, yellow);H 3 (Proceed on sight, 3 yellow lights in V shape)"
+					de.display_values="H 0 (Halt, rot);H 1 (Fahrt, grün);H 1 b (Fahrt, Geschwindigkeitsbeschränkung erwarten, grün blinkend);H 2 (Fahrt, Halt erwarten, gelb);H 3 (Fahrt auf Sicht, 3 gelbe Lichter in Form eines V)"
+					/>
+				<key key="railway:signal:combined:form"
+					value="light" />
+				<combo key="railway:signal:combined:height"
+					text="Signal height"
+					de.text="Signalbauhöhe"
+					values="normal,dwarf"
+					display_values="normal,dwarf"
+					de.display_values="Mastbauform,Zwergsignal"
+					/>
+				<combo key="railway:signal:combined:substitute_signal"
+					text="Substitute Signal"
+					de.text="Auftragssignal"
+					values="DE-VBK:h4"
```

Comment:
JOSM treats the colon as delimiter. Please add `delimiter=";"` to solve this problem.

Reply:
This is fixed now.